cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx

```diff
@@ -50,6 +51,7 @@ function App() {
                             <Route path={NavRoutes.WORKLOAD_CLUSTER_LANDING} element={<WorkloadClusterLanding />}></Route>
                             <Route path={NavRoutes.UNMANAGED_CLUSTER_LANDING} element={<UnmanagedClusterLanding />}></Route>
                             <Route path={NavRoutes.VSPHERE} element={<VSphere />}></Route>
+                            <Route path={NavRoutes.AWS} element={<AwsManagementCluster />} />
```

Comment:
nit: probably good to have consistent <Route> element, either closed /> for all of these, or with <Route></Route> for all of them

Reply:
fixed